pkg/oceancd/model/entities.go

```diff
@@ -119,3 +118,102 @@ func ConvertToClusterDetails(cluster map[string]interface{}) ClusterDetails {
 }
 
 //endregion
+
+type VerificationProviderDetails struct {
+	Name       string `header:"Name"`
+	ClusterIDs string `header:"Cluster ID"`
+	Types      string `header:"Type"`
+	UpdatedAt  string `header:"Updated At"`
+}
+
+func ConvertToVerificationProviderDetails(verificationProvider map[string]interface{}) VerificationProviderDetails {
+	clusterIDs := make([]string, 0)
+	types := make([]string, 0)
+
+	verificationProviderName, _ := verificationProvider["name"].(string)
+	updatedAt, _ := verificationProvider["updatedAt"].(string)
+
+	if ids, ok := verificationProvider["clusterIds"].([]interface{}); ok {
+		for _, id := range ids {
+			if stringID, ok := id.(string); ok {
+				clusterIDs = append(clusterIDs, stringID)
+			}
+		}
+	}
+
+	for verificationProviderType := range verificationProvider {
+		switch verificationProviderType {
+		case Prometheus, Datadog, NewRelic:
+			types = append(types, verificationProviderType)
+		}
+	}
+
+	retVal := VerificationProviderDetails{
+		Name:       verificationProviderName,
+		ClusterIDs: strings.Join(clusterIDs, ", "),
+		Types:      strings.Join(types, ", "),
+		UpdatedAt:  updatedAt,
+	}
+
+	return retVal
+}
+
+type VerificationTemplateDetails struct {
+	Name      string `header:"Name"`
+	Args      string `header:"Arg (Name)"`
+	Metrics   string `header:"Metric (Provider)"`
+	UpdatedAt string `header:"Updated At"`
+}
+
+func ConvertToVerificationTemplateDetails(verificationTemplate map[string]interface{}) VerificationTemplateDetails {
+	argsRes := make([]string, 0)
+	metricsRes := make([]string, 0)
+
+	verificationTemplateName, _ := verificationTemplate["name"].(string)
+	updatedAt, _ := verificationTemplate["updatedAt"].(string)
+
+	if args, ok := verificationTemplate["args"].([]interface{}); ok {
+		for _, arg := range args {
+			if arg, ok := arg.(map[string]interface{}); ok {
+				if name, ok := arg["name"].(string); ok {
+					argsRes = append(argsRes, name)
+				}
+			}
+		}
+	}
+
+	if metrics, ok := verificationTemplate["metrics"].([]interface{}); ok {
```

Comment:
metrics can't be null, can we change this to
`metrics, ok := verificationTemplate["metrics"].([]map[string]interface{})`

Reply:
can we change it to 
`metrics, _ := verificationTemplate["metrics"].([]interface{})`

Reply:
ok leave this validation